ale_linters/elm/make.vim

```diff
@@ -63,14 +73,16 @@ function! ale_linters#elm#make#GetCommand(buffer) abort
     " a sort of flag to tell the compiler not to generate an output file,
     " which is why this is hard coded here.
     " Source: https://github.com/elm-lang/elm-make/blob/master/src/Flags.hs
-    let l:elm_cmd = 'elm-make --report=json --output='.ale#Escape('/dev/null')
+    let l:elm_cmd = ale#Escape(l:elm_exe)
+    \   . ' --report=json'
+    \   . ' --output=' . ale#Escape('/dev/null')
```

Comment:
Use `g:ale#util#nul_file` here instead, as `/dev/null` doesn't work on Windows. (Except for that one Go program...)

Reply:
Done